docs/UsingCppInterOp.rst

```diff
@@ -65,35 +65,245 @@ robust (simple function calls, no inheritance, etc.). The goal is to make it as
 close to the compiler API as possible, and each routine should do just one thing.
 that it was designed for.
 
-Further Enhancing the Dynamic/Automatic bindings in CPPYY
-=========================================================
-The main use case for CppInterOp is the CPPYY service. CPPYY is an
-automatic run-time bindings generator for Python and C++, and supports a wide
-range of C++ features (e.g., template instantiation). It operates on demand and
+How cppyy relates to CppInterOp and how it works
+================================================
+
+CppInterOp ties into several `enhancements in cppyy`_, following are a few:
+
+1. **CppInterOp is a cppyy-inspired Library** that enables interoperability
```

Comment:
I'm not sure I agree with the factual statement here - but also, this is still not an "enhancement in cppyy". Perhaps the introduction to what is cppinterop should just be outside of this list?

Reply:
rephrased the whole section from an "**enhancement in cppyy**" stance to "**How cppyy leverages CppInterOp**". Now the section defines CppInterOp only in ways that is useful for cppyy